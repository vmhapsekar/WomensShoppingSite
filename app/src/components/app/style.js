import styled from 'styled-components'

export const AppWrapper = styled.div`
  font-family: "myriad-pro", "Myriad Pro", Myriad, helvetica, arial, sans-serif;
    font-size: 16px;
    line-height: 1.42857;
    height: auto;
    *:focus {
        outline: none !important;
    }
`