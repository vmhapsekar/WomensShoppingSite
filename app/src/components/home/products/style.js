import styled, {css} from 'styled-components'

export const ProductWrapper = styled.div`
  width: 287px;
  height: 320px;
  padding: 30px 5px;
  border: 1px solid grey;
`

export const ProductStatus = styled.div`
  color: white;
  text-align: center;
  min-height: 27px;
  ${props => props.isExclusive && css`
    background-color: red;
  `}
  ${props => props.isSale && css`
    background-color: green;
  `}
`

export const ProductName = styled.div`
  display: table-cell;
  line-height: 30px;
  position: relative;
  vertically-align: bottom;
  left: 0px;
  color: black;
`

export const ProductPrice = styled.h2`
  display: inline;
  position: relative;
  right: 0px;
  color: black;
`

export const ProductImage = styled.img`
  width: 100%;
  margin: 0 auto;
`
