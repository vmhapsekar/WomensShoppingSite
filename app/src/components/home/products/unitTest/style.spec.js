import React from 'react'
import {ProductStatus} from '../style'
import { shallowToJson } from 'enzyme-to-json'
import { shallow } from 'enzyme'
const products = require('../../../../assets/toolkit/data/products.json')

describe('<ProductStatus />', () => {
  test('Product with Exclusive Status', () => {
    const component = shallow(<ProductStatus isSale={true} />)
    expect(component).toBeDefined()
    expect(shallowToJson(component)).toMatchSnapshot()
  })
  test('Product with Exclusive Status', () => {
    const component = shallow(<ProductStatus isExclusive={true} />)
    expect(component).toBeDefined()
    expect(shallowToJson(component)).toMatchSnapshot()
  })
})
