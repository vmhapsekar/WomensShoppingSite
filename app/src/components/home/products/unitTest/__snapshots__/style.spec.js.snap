// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProductStatus /> Product with Exclusive Status 1`] = `
<div
  className="sc-bwzfXH bHkiij"
/>
`;

exports[`<ProductStatus /> Product with Exclusive Status 2`] = `
<div
  className="sc-bwzfXH iaKjwh"
/>
`;
